app/uk/gov/hmrc/customs/inventorylinking/export/model/actionbuilders/actionBuildersModel.scala

```diff
@@ -94,8 +103,11 @@ trait HasConversationId {
   val conversationId: ConversationId
 }
 
-trait ExtractedHeaders {
+trait HasApiVersion {
   val requestedApiVersion: ApiVersion
+}
+
+trait ExtractedHeaders {
```

Comment:
Guess we can call this `HasClientId` now - but doens't quite fit with the other traits, might need a rethink about how we change `ExtractedHeadersImpl` across all services.

Reply:
Agree. That thought occurred to me. A small piece of tech debt at the moment.